nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -194,7 +229,18 @@ in {
     environment.sessionVariables.LD_LIBRARY_PATH =
       lib.optional cfg.jack.enable "${cfg.package.jack}/lib";
 
+    users = mkIf cfg.systemWide {
+      users.pipewire = {
+        uid = config.ids.uids.pipewire;
+        group = "audio";
+        extraGroups = optional config.security.rtkit.enable "rtkit";
```

Comment:
Please use `lib.` prefix. The `with` should really be removed as it makes everything confusing.

Reply:
Alright, I was unsure as both was used in this module, and not using `lib` occurred more often :)